docs/querying/sql-data-types.md

```diff
@@ -152,14 +152,14 @@ values are treated as zeroes. This was the default prior to Druid 28.0.0.
 
 ## Boolean logic
 
-The [`druid.expressions.useStrictBooleans`](../configuration/index.md#expression-processing-configurations)
-runtime property controls Druid's boolean logic mode. For the most SQL compliant behavior, set this to `true` (the default).
+By default, Druid uses [SQL three-valued logic](https://en.wikipedia.org/wiki/Three-valued_logic#SQL) for filter processing
```

Comment:
Hmm, we should add mention about this parameter to `querying/filters.md` too. This doc is all about SQL, but the same issue affects filters in native queries.

Reply:
add link to here from filters.md